guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -14,8 +14,18 @@ The following updates describe the latest improvements to the `{{site.data.var.c
 
 The release notes include:
 
--   {:.new}New features
--   {:.fix}Fixes and improvements
+-  {:.new}New features
+-  {:.fix}Fixes and improvements
+
+## v2002.0.21
+
+-  {:.new}<!-- MAGECLOUD-3535 -->Added version compatibility validation between Magento version and installed services, such as RabbitMq, Redis, and DB. Displays a warning message if the service version is not compatible.
+
+-  {:.fix}<!-- MAGECLOUD-3806 -->Fixed an issue when running the 'setup:upgrade` command that did not stop the deployment process if there was a failure during the `app:config:import` task.
```

Comment:
```suggestion
```suggestion
-  {:.fix}<!-- MAGECLOUD-3806 -->Fixed an issue when running the `setup:upgrade" command  to ensure that the deployment process stops if a failure occurs during the `app:config:import` task.  Previously, the deployment continue even if a failure occurred. 
```
```